### PR TITLE
ChangedCalculateGenotypePosteriorsDoc

### PR DESCRIPTION
**Pull request by chandrans**
Made minor changes to make easier to read/understand.

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java [hunk 1]
```diff
@@ -23,19 +24,19 @@
...
- * Calculate genotype posterior probabilities given panel data
+ * Calculate genotype posterior probabilities given family genotypes or known population genotypes
  *
  * <p>
- * Given a VCF with genotype likelihoods from the HaplotypeCaller, UnifiedGenotyper, or another source which provides
- * <b>unbiased</b> genotype likelihoods, calculate the posterior genotype state and probability given allele frequency
- * information from both the samples themselves and input VCFs describing allele frequencies in related populations.</p>
+ * Given a VCF with genotype likelihoods from HaplotypeCaller, UnifiedGenotyper, or another source which provides
+ * <b>unbiased</b> genotype likelihoods, calculates the posterior genotype probability given genotype likelihoods
+ * from both the samples themselves and input VCFs describing allele frequencies in related populations.</p>
```
Comment (ldgauthier) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java]: Technically the input VCFs are optional.

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java [hunk 2]
```diff
@@ -23,19 +24,19 @@
...
  *
  * <p>The AF field will not be used in this calculation as it does not provide a way to estimate the confidence interval
  * or uncertainty around the allele frequency, while AN provides this necessary information. This uncertainty is
  * modeled by a Dirichlet distribution: that is, the frequency is known up to a Dirichlet distribution with
  * parameters AC1+q,AC2+q,...,(AN-AC1-AC2-...)+q, where "q" is the global frequency prior (typically q << 1). The
  * genotype priors applied then follow a Dirichlet-Multinomial distribution, where 2 alleles per sample are drawn
- * independently. This assumption of independent draws is the assumption Hardy-Weinberg Equilibrium. Thus, HWE is
+ * independently. This assumption of independent draws is assuming Hardy-Weinberg Equilibrium. Thus, HWE is
```
Comment (ldgauthier) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java]: I would say "This assumption of independent draws follows from the assumption of Hardy-Weinberg equilibrium."

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java [hunk 3]
```diff
@@ -80,50 +81,45 @@
...
- *   -V NA12878.wgs.HC.vcf \
+ * ./gatk-launch CalculateGenotypePosteriors \
+ *   -V NA12878.vcf \
  *   -supporting 1000G_EUR.genotypes.combined.vcf \
- *   -O NA12878.wgs.HC.posteriors.vcf
+ *   -O NA12878.CalculateGenotypePosteriors1000GEuropean.vcf
  * </pre>
  *
- * <h4>Refine the genotypes of a large panel based on the discovered allele frequency</h4>
+ * <h4>Refine the genotypes of a large panel based on the discovered allele frequency in the input VCF</h4>
```
Comment (ldgauthier) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java]: Maybe change "a large panel" to "many samples"?

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java [hunk 4]
```diff
@@ -134,6 +130,7 @@
         oneLineSummary = "Calculate genotype posterior probabilities given panel data",
         programGroup = VariantProgramGroup.class
 )
+@DocumentedFeature
```
Comment (ldgauthier) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java]: Are we still doing the categories in GATK4?  This was in Variant Discovery in GATK3.

**Comment by vdauwera** [comment]
@ldgauthier Could you please review again? I added a few refinements.

**Pull request merged**